frontend/src/components/MusicPage.js

```diff
@@ -9,24 +9,50 @@ const MusicPage = () => {
       setIsLoading(false);
     }, 1500);
   })
-  if(isLoading){
-    return(
+  if (isLoading) {
+    return (
       <Loading />
     );
   }
-    return (
-      <div className="mainContainer">
+
+  const get_SearchResults = e => {
+    const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        searchStr: e,
+      }),
+    };
+    fetch("http://127.0.0.1:8000/spotify/searchAPI", requestOptions)
+      .then((response) => response.json())
+      .then((data) => {
+        //to get a single item within the 'albums' list
+        console.log(data['albums']['items'][0])
+
+        //to get all items within the 'albums' list
+        console.log(data['albums']['items'][0])
```

Comment:
line 33 is supposed to be - console.log(data['albums']['items']), I will fix this

Reply:
Besides this I think this makes sense! Looks good